src/core_plugins/kibana/public/management/sections/indices/_create.html

```diff
@@ -38,18 +34,18 @@
               <input
                 ng-model="index.isTimeBased"
                 type="checkbox">
-              Index contains time-based events
+              <span translate="KIBANA-CONTAINS_TIME_BASED_EVENTS_CHECKBOX"></span>
             </label>
           </div>
 
           <div class="form-group" ng-if="index.isTimeBased">
             <label>
-              Time-field name
+              <span translate="KIBANA-TIME_FIELD_NAME"></span>
               &nbsp;
               <kbn-info info="This field will be used to filter events with the global time filter"></kbn-info>
```

Comment:
Sure, done.

@Bargs, I was unfortunately trying to get the format working you suggested; the `translate-attr` syntax? https://angular-translate.github.io/docs/#/guide/05_using-translate-directive#using-translate-directive_translate-html-attributes 
For example,
<kbn-info translate-attr="{ info: 'KIBANA-FIELD_FILTER_EVENTS_GLOBAL_TIME'}"></kbn-info>
I haven't succeeded yet and there are some postings on it as not working as expected: https://github.com/angular-translate/angular-translate/issues/1577

Is this syntax ok or do we need `translate-attr` ?